ci/after-script.sh

```diff
@@ -5,11 +5,16 @@
 # relative to the root directory of the test container (and should
 # start with "/test/..."), we have to replace this with the path of
 # the service.
-if [ -n "$SERVICE_TEST" ]; then
```

Comment:
I think `if [ -n "${SERVICE_TEST}" ] && [ -n "${TRAVIS_NODE_VERSION}" ]; then ...` works but functions are nice too!

Reply:
Yeah did it like that since Python will be under there soon too and they run under the same check (also the python command is longer since the npm-based coveralls cli accepts it from stdin unlikes pip-based coveralls).